lms/djangoapps/verify_student/views.py

```diff
@@ -1330,4 +1329,12 @@ def _redirect_no_initial_verification(self, user, course_key):
             u"for the course %s.",
             user.id, course_key
         )
-        return redirect(reverse('verify_student_verify_later', kwargs={'course_id': unicode(course_key)}))
+        return redirect(reverse('verify_student_verify_now', kwargs={'course_id': unicode(course_key)}))
+
+
+class VerifyLaterView(RedirectView):
+    """ This view has been deprecated and should redirect to the unified verification flow. """
+    permanent = True
+
+    def get_redirect_url(self, **kwargs):
+        return reverse('verify_student_verify_now', kwargs={'course_id': unicode(kwargs['course_id'])})
```

Comment:
if `'course_id'` is expected in kwargs, could it just be an explicit keyword arg to the view?  (if not, are you concerned about a KeyError here?)